pkg/controller/vsphere/actuator.go

```diff
@@ -103,6 +111,13 @@ func (a *Actuator) Update(ctx context.Context, machine *machinev1.Machine) error
 		return a.handleMachineError(machine, fmt.Errorf(fmtErr), updateEventAction)
 	}
 	if err := newReconciler(scope).update(); err != nil {
+		metrics.RegisterFailedInstanceUpdate(&metrics.MachineLabels{
```

Comment:
This metric is probably pretty useful.  I don't think updates should be failing that frequently.